src/components/structures/RoomView.tsx

```diff
@@ -1137,10 +1137,11 @@ export default class RoomView extends React.Component<IProps, IState> {
         ev.stopPropagation();
         ev.preventDefault();
 
-        this.setState({
-            dragCounter: this.state.dragCounter + 1,
-            draggingFile: true,
-        });
+        this.setState({dragCounter: this.state.dragCounter + 1});
```

Comment:
should we be incrementing the counter if we're not about to be dragging a file? A comment would be good for this